screen/reports/src/main/java/com/ivy/reports/ReportViewModel.kt

```diff
@@ -389,7 +389,24 @@ class ReportViewModel @Inject constructor(
             transactionRepository.findAll()
         }
 
+        val exacludeableByTagTransactionsIds = if (filter.excludedTags.isNotEmpty()) {
```

Comment:
typo: `exacludeableByTagTransactionsIds` to `excludeableByTagTransactionsIds`

Reply:
done